python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -139,7 +139,9 @@ def compile(model: nn.Module,
                 scheduler: _LRScheduler = None,
                 metrics: List[Metric] = None,
                 onnx: bool = False,
-                quantize: bool = False):
+                quantize: bool = False,
+                use_ipex: bool = False,
```

Comment:
talked to Jason. Let's move `use_ipex` and `enable_bf16` back to `Trainer`'s constructor.